packages/client/modules/teamDashboard/components/AgendaItem/AgendaItem.tsx

```diff
@@ -153,6 +153,13 @@ const AgendaItem = (props: Props) => {
         gotoStageId(stageRes.stage.id)
       }
     }
+    if (viewerId !== facilitatorUserId && isFacilitatorStage) {
```

Comment:
-1 the bug is happening because of the check on line 149: `if (viewerId === facilitatorUserId && isFacilitatorStage) {`. We don't want to only change the stage for the facilitator. 

The lines you've added here are largely repeating the code above, which could be confusing for the next developer reading it.

Instead, let's change the if statement, so:

```
  const handleRemove = () => {
    if (isFacilitatorStage) {
      // navigate to the next best stage. onward!
      const stageRes = findStageAfterId(phases, facilitatorStageId!)
      if (gotoStageId && stageRes) {
        gotoStageId(stageRes.stage.id)
      }
    }
    RemoveAgendaItemMutation(atmosphere, {agendaItemId}, {meetingId})
  }
```